lib/IRGen/GenValueWitness.cpp

```diff
@@ -1288,6 +1291,10 @@ FixedPacking TypeInfo::getFixedPacking(IRGenModule &IGM) const {
   if (!fixedTI)
     return FixedPacking::Dynamic;
 
+  // By convention we don't store bitwise takable values inline.
```

Comment:
Is this comment wrong? We don't store *non*-bitwise takable values inline right?

Reply:
Yes. The comment is inverted.